File: Python/Composite_Number.py
print("Composite Number Checker\n")
n = int(input("Enter the number : "))
flag = False
for i in range(2, n//2+1) :
	if n%i == 0:
		flag = True
		break
if(flag):
	print(f"{n} is a composite number.")
else :
	print(f"{n} is not a composite number.")

File: Java/src/Composite_Number.java
/**
 * Composite Number Checker
 * 
 * @author Jaysmito Mukherjee 
 * @version 2019-A-004
 */
import java.util.*;
public class Composite_Number
{
    public static void main()
    {
        System.out.print("\f");
        Scanner sc=new Scanner(System.in);
        System.out.print("Composite Number Checker\n~~~~~~~~~ ~~~~~~ ~~~~~~~\nEnter the number: ");
        int n=sc.nextInt();
        boolean flag=false;
        for(int i=2;i<=n/2;i++)
        {
            if(n%i==0)
            {
                flag=true;
                break;
            }
        }
        if(flag)
            System.out.print(n+" is a Composite number.");
        else
            System.out.print(n+" is not a Composite number.");        
    }
}
